climada/hazard/centroids/test/test_centr.py

```diff
@@ -134,20 +135,48 @@ def test_from_csv(self):
         self.assertEqual(centroids.coord[0][1], 32.57)
         self.assertEqual(centroids.coord[n_centroids - 1][0], -16.23)
         self.assertEqual(centroids.coord[n_centroids - 1][1], 39.91)
+        self.assertEqual(centroids.crs, 'epsg:4326')
 
         Path(tmpfile).unlink()
     
     def test_from_excel(self):
```

Comment:
@sarah-hlsn : there should also be a test for when the data is written and read with climada. So, please write another test which does not use `pd.write_excel`, but `centroids.write_excel`. Then the `from_excel` should work with only using the filename.

Reply:
I may be mistaken but I don't think there is a centroids.write_excel method?

Reply:
Correct. This is missing :). I thought that these were already provided by you.

Reply:
@chahank Following the reasoning of the back and forth compatibility between the read and write methods, I am thinking the output files of the write methods should probably not contain the geometry column of the centroids gdf, but only lat, lon and crs columns (plus potentially on_land, region_id). Correct?

Reply:
Or should we just keep the geometry columnn anyways since it won't be imported when reading the file?

Reply:
hmmm what would be the use to save the geometry column? Can this be read by any means? If there is no use-case, I would not save it.